llpc/util/llpcGfxRegHelper.cpp

```diff
@@ -0,0 +1,111 @@
+/*
+ ***********************************************************************************************************************
+ *
+ *  Copyright (c) 2019-2020 Advanced Micro Devices, Inc. All Rights Reserved.
```

Comment:
I believe we should only put one year in the copyright message, the year of creation.

Reply:
Updated to 2020 only